src/server/system_services/pool_server.js

```diff
@@ -257,6 +257,11 @@ async function create_namespace_resource(req) {
             dbg.error(`fs path ${already_used_by.nsfs_config.fs_path} already exported by ${already_used_by.name}`);
             throw new RpcError('IN_USE', 'Target already in use');
         }
+        const backend_values = ['CEPH_FS', 'NSFv4', 'GPFS'];
```

Comment:
I don't think that's necessary - json schema will do that for you anyway because you defined fs_backend as enum with these options...

Reply:
sure, but I wanted to throw an error that is better understood than INVALID_SCHEMA_DB

Reply:
But your code will not even get executed. I don't think it's needed. 
Bad schema is easy to troubleshoot as it is, and if not we should make it so.

Reply:
ok deleted